### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by JohnSteck9** [ita-social-projects/GreenCity#3448]
Green City

- [x] #3486
   Fix skip-to-main-content-link
   this link should be the first focused link on the page.

- [x] #3490
   Сheck tags for the presence of relevant attributes: 
   roles, name, tabindex ,alternate texts, adaptation of drop-down lists for screen 
   reader.

- [x] #3540
implementation accessibility for links or important headers eko-news page.

- [x] #3589
Make the modal registration window on the site easy to read and control for the screen reader

- [ ] #3588
make content available for screen reader

- [ ] #3487
   identify important regions on the page as logical blocks: 
   Header,Main-content, Footer.
   [https://www.w3.org/TR/wai-aria-practices/examples/landmarks/main.html](url).

Ubs-user
 
- [x] #3497
   Implement on the ubs-user page
   Place the elemet in a free space of content, 
   also the link should be the first focused element 
   on the page.
